matlab/test/arrow/array/tListArray.m

```diff
@@ -0,0 +1,165 @@
+%TLISTARRAY Tests for arrow.array.ListArray
+
+% Licensed to the Apache Software Foundation (ASF) under one or more
+% contributor license agreements.  See the NOTICE file distributed with
+% this work for additional information regarding copyright ownership.
+% The ASF licenses this file to you under the Apache License, Version
+% 2.0 (the "License"); you may not use this file except in compliance
+% with the License.  You may obtain a copy of the License at
+%
+%   http://www.apache.org/licenses/LICENSE-2.0
+%
+% Unless required by applicable law or agreed to in writing, software
+% distributed under the License is distributed on an "AS IS" BASIS,
+% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+% implied.  See the License for the specific language governing
+% permissions and limitations under the License.
+
+classdef tListArray < matlab.unittest.TestCase
+
+    properties (Constant)
+        Traits = arrow.type.traits.traits(arrow.type.ID.List)
+    end
+
+    properties (TestParameter)
+        TestArrowArray
+    end
+
+    methods (TestParameterDefinition, Static)
+
+        function TestArrowArray = initializeTestArrowArray()
+            %% Empty (zero-element) list
+            Type = arrow.list(arrow.float64());
+            NumElements = int64(0);
+            Valid = logical.empty(0, 1);
+            Offsets = arrow.array(int32(0));
+            Values = arrow.array([]);
+            ArrowArray = arrow.array.ListArray.fromArrays(Offsets, Values, Valid=Valid);
+            MatlabArray = {cell.empty(0, 1)};
+
+            TestArrowArray.EmptyList = struct( ...
+                ArrowArray=ArrowArray, ...
+                MatlabArray=MatlabArray, ...
+                Properties=struct(...
+                    Type=Type, ...
+                    NumElements=NumElements, ...
+                    Valid=Valid, ...
+                    Offsets=Offsets, ...
+                    Values=Values ...
+                ) ...
+            );
+
+            %% List with NULLs
```

Comment:
Could you mention the list type here too?

Reply:
Fixed.